### ISSUE TITLE
Texas data

### ISSUE THREAD

**Issue by lilianmarey** [timriffe/covid_age#38]
Hello, I have been working on the dataset (I'm conding a Dash application n Python to visualize the database).
I noticed that data from Texas was false from 2020/07/29 (values are too high).
![Capture](https://user-images.githubusercontent.com/68290011/91466798-ecdb8c00-e88f-11ea-9487-b529563462ed.PNG)

**Comment by timriffe** [comment]
Thanks for reporting this. I've fixed the automatic capture script for Texas, which had a tiny but high leverage bug. I've gone ahead and made another bigger change to this series- moving forward total cases and deaths will be revised in retrospect with each update. The output should be corrected in about 24 hours. 
I'll close this issue after verifying the output tomorrow.
Will be interested to see your dashboard implementation, thanks for doing that!

**Comment by timriffe** [comment]
This is now fixed, thanks again for reporting, and please send a link to your dash when you're ready to!

**Comment by lilianmarey** [comment]
You are welcome !
I have been working with Marius Pascariu during an internship in August.
My app aims at illustrating the database with several visualization tools (charts, histograms, maps), and also computing relevant metrics from native ones. 
I will be glad to share it with you, the thing is the app wont probably be available by a public link (it will be on Scor's servers, the company where I did my internship).
However I can share the code, I have to make some corrections but it will be ready soon. I will notify you when it's ready ;)
(Also, I did it in Pyhton, I know that you guys use a lot of R, I hope it won't be too annoying for you)
